packages/contracts-bedrock/.storage-layout

```diff
@@ -152,6 +150,12 @@
 | hash           | bytes32 | 2    | 0      | 32    | contracts/L2/L1Block.sol:L1Block |
 |----------------+---------+------+--------+-------+----------------------------------|
 | sequenceNumber | uint64  | 3    | 0      | 8     | contracts/L2/L1Block.sol:L1Block |
+|----------------+---------+------+--------+-------+----------------------------------|
+| batcher        | address | 3    | 8      | 20    | contracts/L2/L1Block.sol:L1Block |
+|----------------+---------+------+--------+-------+----------------------------------|
+| l1FeeOverhead  | uint256 | 4    | 0      | 32    | contracts/L2/L1Block.sol:L1Block |
+|----------------+---------+------+--------+-------+----------------------------------|
+| l1FeeScalar    | uint256 | 5    | 0      | 32    | contracts/L2/L1Block.sol:L1Block |
 +----------------+---------+------+--------+-------+----------------------------------+
 
 =======================
```

Comment:
I wonder why the `SystemConfig` contract wasn't added to this file?

Reply:
I believe that we have a list of contracts to be included, its not just every contract

Reply:
oh, that sounds right. ALso something we can address in a subsequent PR.

Reply:
Will fix